pytorch_lightning/overrides/torch_distributed.py

```diff
@@ -53,13 +54,21 @@ def _broadcast_object_list(object_list, src=0, group=None):
 
     group_backend = get_backend(group)
     is_nccl_backend = group_backend == Backend.NCCL
+    import os
+
+    dist_backend = os.environ.get("PL_TORCH_DISTRIBUTED_BACKEND")
+    is_hcl_backend = group_backend == torch.distributed.Backend(str(dist_backend))
```

Comment:
All the code in this file will be removed in # together with the support for PyTorch 1.6